suite-common/wallet-core/src/send/sendFormActions.ts

```diff
@@ -28,15 +28,15 @@ const storePrecomposedTransaction = createAction(
     `${SEND_MODULE_PREFIX}/store-precomposed-transaction`,
     (payload: {
         formState: FormState;
-        transactionInfo: PrecomposedTransactionFinal | TxFinalCardano;
+        precomposedTransaction: GeneralPrecomposedTransactionFinal;
     }) => ({
         payload,
     }),
 );
 
 const storeSignedTransaction = createAction(
     `${SEND_MODULE_PREFIX}/store-signed-transaction`,
-    (payload: FormSignedTx) => ({
+    (payload: { serializedTx: SerializedTx; signedTx?: BlockbookTransaction }) => ({
```

Comment:
will it work if the txn is from nonblockbook blockchains like solana, ripple or cardano?

Reply:
Yes, `signedTx` is optional argument. Only BTC networks utilize it.